docs/docs/en/guide/task/mlflow.md

```diff
@@ -5,77 +5,140 @@
 [MLflow](https://mlflow.org) is an excellent open source platform to manage the ML lifecycle, including experimentation,
 reproducibility, deployment, and a central model registry.
 
-Mlflow task is used to perform mlflow project tasks, which include basic algorithmic and autoML capabilities (
-User-defined MLFlow project task execution will be supported in the near future)
+MLflow task plugin used to execute MLflow tasks，Currently contains Mlflow Projects and MLflow Models.（Model Registry will soon be rewarded for support）
+
+- Mlflow Projects: Package data science code in a format to reproduce runs on any platform.
+- MLflow Models: Deploy machine learning models in diverse serving environments.
+- Model Registry: Store, annotate, discover, and manage models in a central repository.
+
+The Mlflow plugin currently supports and will support the following:
+
+- [ ] MLflow Projects
+    - [x] BasicAlgorithm: contains lr, svm, lightgbm, xgboost
+    - [x] AutoML: AutoML tool，contains autosklean, flaml
+    - [ ] Custom projects: Support for running your own MLflow projects
+- [ ] MLflow Models
+    - [x] MLFLOW: Use `MLflow models serve` to deploy a model service
+    - [x] Docker: Run the container after packaging the docker image
+    - [ ] Docker Compose: Use docker compose to run the container, Will replace the docker run above
+    - [ ] Seldon core: Use Selcon core to deploy model to k8s cluster
+    - [ ] k8s: Deploy containers directly to K8S 
+    - [ ] mlflow deployments: Built-in deployment modules, such as built-in deployment to SageMaker, etc
+- [ ] Model Registry
+    - [ ] Register Model: Allows artifacts (Including model and related parameters, indicators) to be registered directly into the model center
```

Comment:
It is a document for users, I think we should not add something we do not support yet, it is for developers or contributors, maybe we should only add what we support here, but I do not have strong opinions here.

Reply:
I think a list of future supported components is not only shown for developers but users. This can attract users by showing them future features.